### PR TITLE
www: add frontmatter and meta description handling

### LINKED ISSUE #165: www: add meta description to docs page
There is no `<meta name="description" />` on the docs pages. One should be added. The descriptions should be manually added for each docs page in the front matter (I will do this). I don't want automatic generation of the descriptions based on content - it never contains the right info that way.

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/lucacasonato/fresh/issues/165

### CODE REVIEW

**Comment by lucacasonato** [comment]
It looks like the docs pages are crashing with this change: https://fresh-wawt210e3rw0.deno.dev/docs/introduction.

**Comment by bartlomieju** [comment]
Fixed now

File: www/routes/docs/[...slug].tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ interface Data {
 
 interface Page extends TableOfContentsEntry {
   markdown: string;
+  frontMatterData: Record<string, string>;
```
Comment (lucacasonato) [inline www/routes/docs/[...slug].tsx]: Can you rename to just `data`?

Also should really be `unknown` or `Record<string, unknown>` as yaml can have non string values. This is not too important tho.

**Review by lucacasonato — APPROVED** [review]
Thanks Bartek! :tada:

**Pull request merged**